tests/by-util/test_cksum.rs

```diff
@@ -148,7 +148,7 @@ fn test_tag_after_untagged() {
         .arg("-a=md5")
         .arg("lorem_ipsum.txt")
         .succeeds()
-        .stdout_is_fixture("md5_single_file.expected");
+        .stdout_is("cd724690f7dc61775dfac400a71f2caa  lorem_ipsum.txt\n");
```

Comment:
this is caused by the tag / untagged change

Reply:
I have to admit I don't understand this change even with your comment :| With GNU `cksum`, I get the following output format:
```
$ cksum --untagged --tag --algo=md5 f
MD5 (f) = 14758f1afd44c09b7992073ccf00b43d
```

Reply:
right, the order matters...